client/foundries_tuf_root.go

```diff
@@ -83,6 +87,15 @@ func (a *Api) TufProdRootPost(factory string, root []byte) (string, error) {
 	return a.tufRootPost(factory, true, root)
 }
 
+func (a *Api) TufRootUpdatesInit(factory string) (res TufRootUpdatesInit, err error) {
+	var body *[]byte
+	url := a.serverUrl + "/ota/repo/" + factory + "/api/v1/user_repo/root/updates"
+	if body, err = a.Post(url, nil); err == nil {
+		err = json.Unmarshal(*body, &res)
```

Comment:
maybe your commits got messed up when cleaning up. I don't see `res` being defined here?

Reply:
it is defined in the function signature